guillotina/contrib/dbusers/content/groups.py

```diff
@@ -0,0 +1,57 @@
+from guillotina import configure
+from guillotina import schema
+from guillotina.content import Folder
+from guillotina.contrib.dbusers import _
+from guillotina.interfaces import IFolder
+from zope.interface import implementer
+
+import typing
+
+
+class IGroupManager(IFolder):
+    pass
+
+
+class IGroup(IFolder):
+
+    name = schema.TextLine(title=_("Group name"), required=False)
+    description = schema.TextLine(title=_("Group Description"), required=False)
+    user_roles = schema.List(
+        title=_("Roles"), value_type=schema.TextLine(), required=False
+    )
```

Comment:
Are those supposed to be actual guillotina global roles?
If so, don't we need to change the core security logic to conditionally (if dbusers is installed) check for this aswell?

Reply:
not needed. 
Guillotina already checks permissions assigned to users' groups

Reply:
But ferran, these are global roles a user gets assigned when authenticating against a container.

Think of this as what the roles an oauth/user api server would be returning for a user.

Reply:
Then again, I see your point. We could assign everything at the root of the container and get the same affect.